docusaurus.config.js

```diff
@@ -63,22 +68,14 @@ module.exports = {
               to: '/',
             },
             {
-              label: 'Performance',
-              to: '/performance/',
-            },
-            {
-              label: 'Security',
-              to: '/security/',
+              label: 'API Reference',
```

Comment:
What do you think of bubbling up the perf and security links somehow?
From the latest survey they were a lot of devs that weren't aware of these lists and we want to make sure they have them in mind.

Reply:
@molant I feel like bubbling it up somehow is a good idea, but the current footer content felt a bit jarring since these made up 2/3 items in the Docs section 🤔

Reply:
We can always add more links if we think those are relevant.
My main concern is that we are duplicating the links that are already in the header that are already visible and from what I'm seeing in the stats the perf one is driving some traffic.



Ideally we should add some analytics to see how many people click

Reply:
addressed in e6ed369